### ISSUE TITLE
How does this work in Angular 1.4?

### ISSUE THREAD

**Issue by mattiLeBlanc** [meteor-typescript/meteor-typescript-libs#44]
Hi, 

I am currently building in a Yo Gulp Angular stack with webpack and I am using Typescript and code angular in 2.0 style.
I tried today to port my code to Meteor + Angular + Typescript but I ran into a couple of issues and question marks.
And I couldn't find any example apps.
Anyone having suggestions or examples to get me started?

**Comment by fullflavedave** [comment]
I haven't worked with Meteor + Angular + TypeScript before, but the angular-meteor definitions might work better for you:  https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/angular-meteor

**Comment by mattiLeBlanc** [comment]
Yes thank you. I found those since I started digging yesterday. With TSD I can install all the definitions. Works like a charm.
